kateryna_kravchuk/task_03.md

```
@@ -0,0 +1,243 @@
+`tar -czf kr.tar.gz kr.txt`  
+`ls -l`   
+```
+total 4
+lrwxrwxrwx. 1 kate_kr kate_kr   6 Feb 14 20:17 kr.lnk -> kr.txt
+-rw-r--r--. 1 kate_kr kate_kr 116 Feb 14 20:23 kr.tar.gz
+-rw-r--r--. 1 kate_kr kate_kr   0 Feb 14 20:17 kr.txt
+```  
+
+---
+
+#### 11. Use the du command to analyze the disk usage of your home directory, then display the top 5 largest files or directories.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)